let-your-reader-connect-to-your-subject/readme.md

```diff
@@ -0,0 +1,28 @@
+> Goals of this guide: To let your audience connect to the protagonist of your story.
+
+# Letting the reader connect to your subject
+
+It is quite hard to find information on how you can stimulate a connection between the reader and the protagonist.
+
+We did find quite some stuff on how to make your main character in a fictional story approachable.
+
+In theory you could say that the principles are the same, only now you have to use the information you have instead of making something up.
+
+## Tips from the internet
+
+We rewrote tips from Writersrelief.com to make readers care about your characters, in an attempt to make them more suitable for our purpose. You can either find someone to interview or write about that has some of these traits, or look for them wile interviewing and researching.
+
```

Comment:
```You can either find someone to interview or write about that has some of these traits, or look for them wile interviewing and researching.```

I don't get the difference. People posses traits of acquire them, but not by you looking for them right?